docs/v1/user-story.md

```diff
@@ -0,0 +1,124 @@
+# A Year In The Life Of A Kubernetes Service Developer
+
+## 0. Today
+I'ma developer in an enterprise environment.  For some fun in my spare time, I’m building a node.js chat application for me 
+and a couple of coworkers to goof around with.  It’s really simple.  It runs on one container.  To get going, I simply 
+download a node.js container onto minikube on my laptop and started coding.  I want some co-workers to be able to find the 
```

Comment:
I kind of think that the rest of this section, starting with "I want some co-workers..." until the end of the paragraph, should be another section.  I think that section should describe in more detail how registering this service in the catalog means that there is a stable endpoint for its consumers.